components/hooks/useTimer/components/useTimer.js

```diff
@@ -54,7 +54,8 @@ export function useTimer ({ autoStart = true, interval = 1000, startTime, endTim
         .hours()
         .toString()
         .padStart(2, '0'),
-      days: parseInt(temp.asDays(), 10)
+      days: parseInt(temp.asDays(), 10),
```

Comment:
@majid I do not think we need to shows days on the timer - just hours - could be 1000 hours but thats fine, please amend

Reply:
Done